src/utils/whitelistApis.ts

```diff
@@ -2833,6 +2833,20 @@ export const API_LIST = {
                 ROLE.MDO_ADMIN,
              ],
          },
+         '/proxies/v8/workflow/admin/transition/bulkupdate': {
+            checksNeeded: [CHECK.ROLE],
+            // tslint:disable-next-line: object-literal-sort-keys
+            ROLE_CHECK: [
+                ROLE.MDO_ADMIN,
+             ],
+         },
+         '/proxies/v8/workflow/admin/bulkuploadfile/download/:fileName': {
+            checksNeeded: [CHECK.ROLE],
+            // tslint:disable-next-line: object-literal-sort-keys
+            ROLE_CHECK: [
+                ROLE.MDO_ADMIN,
```

Comment:
MDO_LEADER also should have these functionalities... add that role too.

Reply:
This is done.